closure/goog/array/array.js

```diff
@@ -700,6 +700,24 @@ goog.array.remove = function(arr, obj) {
 
 
 /**
+ * Removes the last occurrence of a particular value from an array.
+ * @param {!Array<T>|!goog.array.ArrayLike} arr Array from which to remove
+ *     value.
+ * @param {T} obj Object to remove.
+ * @return {boolean} True if an element was removed.
+ * @template T
+ */
+goog.array.removeLast = function(arr, obj) {
+  var i = goog.array.lastIndexOf(arr, obj);
+  var rv;
```

Comment:
Might as well just set rv = (i >= 0) here, where it's more straightforward.  Could also use a better name (like isPresent).  Alternatively, don't even bother setting the variable.

    if (i >= 0) {
      // remove
      return true;
    }
    return false;